src/input/InputDriverManager.cc

```diff
@@ -132,7 +132,9 @@ std::string InputDriverManager::listDrivers() const
 
 void InputDriverManager::closeDrivers()
 {
-    timer->stop();
+    if (timer != nullptr) {
+        timer->stop();
+    }
     InputEventMapper::instance()->stop();
```

Comment:
Thanks, I guess need to review the remaining part of this method too. Looks like InputEventMapper creates a QTimer in the constructor.